yt_dlp/extractor/theplatform.py

```diff
@@ -103,6 +105,36 @@ def _add_chapter(start_time, end_time):
             for chapter in tp_chapters[:-1]:
                 _add_chapter(chapter.get('startTime'), chapter.get('endTime'))
             _add_chapter(tp_chapters[-1].get('startTime'), tp_chapters[-1].get('endTime') or duration)
+        info_keywords_str = info.get('keywords', {str_or_none})
+        tags = []
+        if (info_keywords_str is not None) and (info_keywords_str != ''):
+            tags = re.split(', |; |,', info_keywords_str)
+        location = None
+        series = None
+        season_number = None
```

Comment:
these should be moved down by where the values are extracted, but my other suggestion below will make initializing the variables unnecessary
```suggestion
```

Reply:
Yes, that other proposition is better in total.